.github/actions/install-tools/action.yml

```diff
@@ -0,0 +1,38 @@
+name: Install Tools
+description: Installs Node, Rust, and WASM tools
+
+inputs:
+  token:
+    required: true
+
+runs:
+  using: composite
+
+  steps:
+    - name: Install Tools
```

Comment:
Let's avoid having steps which have the same name as the actions they're contained within. Ideally this should be more specific.